### DISCUSSION TITLE
How to use TurboRepo on Netlify?

### DISCUSSION THREAD

**Discussion by Joroze** [vercel/turborepo#5548] [Help] [answered]
Can't find good documentation around implementing Turbo with Netlify.

**Comment by parasew** [comment, marked as answer]
Did you see the thread [Architecture for PNPM + Turborepo + Multiple Netlify sites in Monorepo](https://answers.netlify.com/t/architecture-for-pnpm-turborepo-multiple-netlify-sites-in-monorepo/69515)
